### PR TITLE
Fix/371 login page

### LINKED ISSUE #371: Fixes to login page
## Current Behavior
  - a registration form is not intuitive with its scroll 
  - sign up is the first option when user enters a login page

## Tasks to do 
- [ ] make header's position fixed, add div with same bootstrap classes and height 100vh and remove line with overflow-y auto on .login class
- [ ] in login-page.ts change default mode as LOG_IN (currently REGISTER)

### PR DESCRIPTION
**Pull request by juliapacula**
Fixes #371   

## Proposed Changes

  - fix overflow on too logn registration form
  - add BEM naming convention for classes
  - make logging in the first action user sees on login page

## Additional comment (optional)

What about deciding on a naming convention and making our good practices in code? I suggested BEM but I am open for something else! :smile: And also I suggest having one fixed order of css (eg. https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ ). I'm curious about your opinion @Darkoon :wink:

### CODE REVIEW

File: frontend/src/app/pages/login-page/login-page.component.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <div class="row">
-    <header class="col-sm-12 col-lg-4">
-        <div class="logo">
-            <img src="assets/icon/medtagger_logo.svg"/>
+    <header class="header col-sm-12 col-lg-4">
+        <div class="header-logo">
+            <img src="../../../assets/icon/medtagger_logo.svg"/>
```
Comment (jszemplinski) [inline frontend/src/app/pages/login-page/login-page.component.html]: Path like that will cause trouble when MedTagger will be run from a subdirectory (mentioned and fixed it in #362), I'd suggest setting it back to `assets/icon/medtagger_logo.svg` ;)

Comment (kolszewska) [reply frontend/src/app/pages/login-page/login-page.component.html]: @jszemplinski is right, I have wanted to write the same thing.

**Comment by kolszewska** [comment]
Hi @juliapacula as @Darkoon mentioned, we could all use some document for this. Could you create an issue for that?

**Comment by kolszewska** [comment]
Hi @Darkoon @juliapacula can we proceed with this? Julia created an issue #380 for documentation.

**Pull request merged**